app/models/work/additional_credit.rb

```diff
@@ -2,8 +2,9 @@ class Work
   class AdditionalCredit
     include AttrJson::Model
 
-    ROLES = ['photographed_by']
+    ROLES = ['photographed_by', 'translator']
     NAMES = [
+      'Jocelyn R. McDaniel',
```

Comment:
should these be ordered in some way? It looks like they were alphabetical by first name (ie by string) before, let's keep it that way?

Reply:
Good catch!